src/framework/audio/internal/platform/lin/linuxaudiodriver.cpp

```diff
@@ -257,6 +257,50 @@ async::Notification LinuxAudioDriver::availableOutputDevicesChanged() const
     return m_availableOutputDevicesChanged;
 }
 
+unsigned int LinuxAudioDriver::outputDeviceSampleRate() const
+{
+    return s_format.sampleRate;
+}
+
+bool LinuxAudioDriver::setOutputDeviceSampleRate(unsigned int sampleRate)
+{
+    if (s_format.samples == sampleRate) {
+        return true;
+    }
+
+    bool reopen = isOpened();
+    close();
+    s_format.sampleRate = sampleRate;
+
+    bool ok = true;
+    if (reopen) {
+        ok = open(s_format, &s_format);
+    }
+
+    if (ok) {
+        m_sampleRateChanged.notify();
+    }
+
+    return ok;
+}
+
+async::Notification LinuxAudioDriver::outputDeviceSampleRateChanged() const
+{
+    return m_sampleRateChanged;
+}
+
+std::vector<unsigned int> LinuxAudioDriver::availableOutputDeviceSampleRates() const
+{
+    // ALSA API is not of any help to get sample rates supported by the driver.
+    // (snd_pcm_hw_params_get_rate_[min|max] will return 1 to 384000 Hz)
+    // So just returning a sensible hard-coded list.
+    return std::vector<unsigned int> {
+        44100,
+        48000,
```

Comment:
Shouldn't we add 88200 to this list?

Reply:
And perhaps 192000 as well? I believe that's not uncommon as well, even my cheap Windows laptop supports it (or at least, gives it as an option...)